ui/src/components/TrafficPage.tsx

```diff
@@ -108,15 +114,18 @@ export const TrafficPage: React.FC<TrafficPageProps> = ({setAnalyzeStatus, onTLS
             const message = JSON.parse(e.data);
             switch (message.messageType) {
                 case "entry":
-                    const entry = message.data
-                    if (!focusedEntryId) setFocusedEntryId(entry.id.toString())
-                    let newEntries = [...entries];
-                    setEntries([...newEntries, entry])
-                    if(listEntry.current) {
-                        if(isScrollable(listEntry.current.firstChild)) {
-                            setDisableScrollList(true)
-                        }
-                    }
+                    const entry = message.data;
+                    if (!focusedEntryId) setFocusedEntryId(entry.id.toString());
+                    setEntriesBuffer([
+                        ...entriesBuffer,
+                        <EntryItem
+                            key={entry.id}
+                            entry={entry}
+                            setFocusedEntryId={setFocusedEntryId}
+                            style={{}}
```

Comment:
can we remove this empty "style"

Reply:
@gadotroee I tried a patch like this:
```diff
diff --git a/ui/src/components/EntryListItem/EntryListItem.tsx b/ui/src/components/EntryListItem/EntryListItem.tsx
index e9346d6..1ff59f6 100644
--- a/ui/src/components/EntryListItem/EntryListItem.tsx
+++ b/ui/src/components/EntryListItem/EntryListItem.tsx
@@ -38,7 +38,7 @@ interface Rules {
 interface EntryProps {
     entry: Entry;
     setFocusedEntryId: (id: string) => void;
-    style: object;
+    style?: object;
     updateQuery: any;
 }
 
diff --git a/ui/src/components/TrafficPage.tsx b/ui/src/components/TrafficPage.tsx
index ae24851..55ec883 100644
--- a/ui/src/components/TrafficPage.tsx
+++ b/ui/src/components/TrafficPage.tsx
@@ -122,7 +122,6 @@ export const TrafficPage: React.FC<TrafficPageProps> = ({setAnalyzeStatus, onTLS
                             key={entry.id}
                             entry={entry}
                             setFocusedEntryId={setFocusedEntryId}
-                            style={{}}
                             updateQuery={updateQuery}
                         />
                     ]);
```

But since that object is populated in here:

https://github.com/mertyildiran/react-scrollable-feed/blob/01c8dd7c031aa438edc58fa6dda24d9c8d9ff0ce/src/index.tsx#L271-L272

It throws this error:

```
Uncaught TypeError: Cannot set properties of undefined (setting 'top')
    at t.render (index.tsx:271)
    at $o (react-dom.production.min.js:187)
    at jo (react-dom.production.min.js:186)
    at Vs (react-dom.production.min.js:269)
    at Il (react-dom.production.min.js:250)
    at Nl (react-dom.production.min.js:250)
    at _l (react-dom.production.min.js:250)
    at hl (react-dom.production.min.js:243)
    at react-dom.production.min.js:123
    at t.unstable_runWithPriority (scheduler.production.min.js:18)
```

If you look at the diff, it was like that before as well:

https://github.com/up9inc/mizu/pull/452/files#diff-54e1660fae70e6338ceb35b4a59835b8271c56969997f71bdfe01fea7cbb6c28L37

Reply:
Ok, I will mark this as resolved